packages/core/src/components/Input/Input.types.tsx

```diff
@@ -8,7 +8,10 @@ export interface InputProps extends Partial<Pick<UseFormReturn, 'register'>> {
   placeholder?: string;
   validationSchema?: Record<string, unknown>;
   disabled?: boolean;
-  append?: React.ReactElement;
+  startElement?: React.ReactElement;
+  startElementPadding?: number;
```

Comment:
I would suggest using `className` instead of `startElementPadding` and `endElementPadding`.  And that pass through `className` to `<InputBase />`, like this: 
```
const Input = ({ className }) => <InputBase className={className} />
```
and then: 
```
<Input css={{ paddingLeft: 40, paddingRight: 40 }} />
```

`className` allows to use Emotion's `css` prop: https://emotion.sh/docs/css-prop#use-the-css-prop

Reply:
Improved